deploy.sh

```
@@ -1,16 +1,18 @@
 #!/usr/bin/bash
 
+PROJECT=TODO
+
 cd $(realpath "$(dirname "${BASH_SOURCE[0]}")")/src &&
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)